### PR TITLE
Switch airfoil url and homepage to https

### PR DESCRIPTION
**Pull request by gtklocker**

### CODE REVIEW

File: Casks/airfoil.rb [hunk 1]
```diff
@@ -2,9 +2,9 @@
   version :latest
   sha256 :no_check
 
-  url 'http://neutral.rogueamoeba.com/mirror/files/Airfoil.zip'
+  url 'https://d2oxtzozd38ts8.cloudfront.net/airfoil/download/Airfoil.zip'
```
Comment (vitorgalvao) [inline Casks/airfoil.rb]: Needs a `# cloudfront.net is the official download host per the vendor homepage` comment above.

Comment (gtklocker) [inline Casks/airfoil.rb]: Fixed

**Comment by vitorgalvao** [comment]
Looks good, could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by gtklocker** [comment]
@vitorgalvao Done.

**Pull request merged**